package.json

```diff
@@ -235,6 +236,8 @@
 		"electron-devtools-installer": "^3.1.1",
 		"electron-notarize": "^1.0.0",
 		"electron-root-path": "^1.0.16",
+		"electron-updater": "^4.3.5",
+		"electron-log": "^4.2.4",
```

Comment:
These must be non-dev dependencies

Reply:
Also moved `async-retry` as non-dev dep, as it was causing not-found error in production.